portal-api/src/v1/roles/validate-user-is-acting-on-self-to-change-password-or-is-admin.js

```diff
@@ -0,0 +1,23 @@
+const axios = require('axios');
+const CONSTANT = require('../../constants');
+/**
+ * Creates a middleware that returns a 401 error if the current user logs with
+ * old password have at least one of the allowedRoles, and calls the next handler otherwise.
+ * @param {{ allowedRoles: string[] }}
+ * @returns {(req, res, next) => void}
+ */
+const validateSelfOrAdmin = (req, res, next) => {
+  if (!req.user) {
+    return res.status(axios.HttpStatus.UNAUTHORIZED).send('Unauthorized');
```

Comment:
Can this please be `HttpStatusCode.Unauthorized` to keep it consistent.

Reply:
Changed to `HttpStatusCode.Unauthorized`